osl/interface_linux.go

```diff
@@ -130,6 +130,28 @@ func (n *networkNamespace) Interfaces() []Interface {
 	return ifaces
 }
 
+func (i *nwIface) SetSecondaryIP(ip *net.IPNet, add bool) error {
```

Comment:
`nwIface.ipAliases` currently is a slice of IPs to be programmed as an alias on creation time. Where as here the requirement is to dynamically add/delete the secondary ip (aka alias) when other services are created or delete on the same network. Also, it doesn't look like we have to store these IPs on every end-point. That is why I took the current approach which is basically a direct manipulation of the osl interface to add or remove the alias IP.